spec/hardware/system.fmf

```diff
@@ -20,6 +20,9 @@ description: |
            # Integer or string, required number of NUMA nodes.
            numa-nodes: 2|">= 2"
 
+           # System management interface
+           management-interface: IPMI
+
     .. versionchanged:: 1.39
```

Comment:
Add also `versionchanged` block for the new constraint:

```
    .. versionchanged:: 1.33
       Added ``location.lab-controller`` into specification.
```

Reply:
added